appinfo/info.xml

```diff
@@ -37,7 +37,7 @@ Before you update to a new version, [check the changelog](https://github.com/nex
         <lib>SimpleXML</lib>
         <lib>iconv</lib>
         <owncloud max-version="0" min-version="0"/>
-        <nextcloud min-version="14" max-version="16"/>
+        <nextcloud min-version="15" max-version="17"/>
```

Comment:
Why is the minimal version bump needed? 14 running on PHP 7.1 would be fine right?

Reply:
It didn't work on 14 I thought that i pushed that commit, but apparently not.

It seems like we lose compatibility with 14, because the app-checker doesn't know about the return statements with a '?'
https://travis-ci.org/nextcloud/news/jobs/519546871

Reply:
That seems more like a server issue that should be backported then

Reply:
Yea you are probably right 😁 can you deal with that or at least get some attention?